documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
```

Comment:
Do we need to distinguish between policy, standards and procedures? Policy and standards, if properly written, should never contain any of the categories of sensitive information set out below. We need to be clear about this - if our policy or standards are sensitive then we've done something wrong. Some of our procedures (e.g. breakglass documentation) might need to contain sensitive information and should therefore live elsewhere.